ovve_ui/utils/out_packet.py

```diff
@@ -52,17 +52,21 @@ def calculate_mode(self, in_mode, run_state) -> int:
         mode = 0
         # get the updates from settings 
         # Set the mode value byte which also includes start bit
-        if in_mode == 0 and run_state == 0:
-            return 0
-        elif in_mode == 0 and run_state == 1:
-            mode =  1
-        elif in_mode == 1 and run_state == 0:
-            mode =  2
-        elif in_mode == 1 and run_state == 1:
-            mode = 3
-        elif in_mode == 2 and run_state == 0:
-            mode = 4
-        elif in_mode == 2 and run_state == 1:
-            mode = 5
+
+        mode = in_mode & 0x7f | (run_state << 7) & 0x80
```

Comment:
```suggestion
        mode = (in_mode & 0x7f) | ((run_state << 7) & 0x80)
```
Even though in python `&` has precedence over `|`, Ive had annoying bugs come from not being explicit with parens, so its best to add them to make operations explicitly ordered.